src/Kernel-CodeModel/BlockClosure.class.st

```diff
@@ -993,6 +1028,40 @@ BlockClosure >> valueWithin: aDuration onTimeout: timeoutBlock [
 				ifFalse:[ e pass]]
 ]
 
+{ #category : 'evaluating' }
+BlockClosure >> valueWithinMilliseconds: milliseconds onTimeout: timeoutBlock [
```

Comment:
I'm not convinced this method should be in Kernel :) But I understand it was before like this already.